### PR TITLE
[ja] Localize Ingress for Japanese

### LINKED ISSUE #2747: [ja] Localize Ingress for Japanese
Localize https://glossary.cncf.io/ingress/

### PR DESCRIPTION
**Pull request by Okabe-Junya**
### Describe your changes

Localize Ingress for Japanese

### Related issue number or link (ex: `resolves #issue-number`)

resolves #2747 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Review by naonishijima — APPROVED** [review]
LGTM

File: content/ja/ingress.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+title: Ingress
+status: Completed
+category: テクノロジー
+tags: ["基礎"]
+---
+
+Ingressは、クラスター内で動作するコンテナあるいはコンテナ群へのインターネットトラフィックを外部から管理するためのルールセットです。
```
Comment (kaitoii11) [inline content/ja/ingress.md:8]: ルールセットを外部から管理する、と読み取れるような気がするので、ちょっと構成変えたほうが良いかもです。

Comment (Okabe-Junya) [reply content/ja/ingress.md]: ありがとうございます。

確かに修飾関係が少しおかしかったですね。

> to manage internet traffic from outside into ...

だったので、「〜〜外部からのトラフィックを管理する」に修正しました

**Review by kaitoii11 — APPROVED** [review]
LGTM

**Pull request merged**